main/faq.py

```diff
@@ -381,5 +381,11 @@ def faq_seed():
 	"""
 	return(df)
 
+# FORUM def faq_forum(): df = """ 
```

Comment:
@cafebedouin new line to separate the comment from the function otherwise it will not work